### DISCUSSION TITLE
Example of using Connections.getRDFCollection for streaming

### DISCUSSION THREAD

**Discussion by toretest** [eclipse-rdf4j/rdf4j#3379] [Q&A] [answered]
Is there an example using

`try(RepositoryConnection conn = rep.getConnection()) {
   Model rdfList = Connections.getRDFCollection(conn, node, new LinkedHashModel());
}`

see also
https://rdf4j.org/documentation/programming/repository/

node 
 @param head               the start resource of the RDF Collection. May not be {@code null}.
 
 Example of this.

**Comment by abrokenjester** [comment, marked as answer]
I'm not sure I understand what you're after. The usage is explained in detail in the link you have in your post. Including examples. Is there something specific that you still have questions about?

**Reply by toretest** [reply]
An example of the "node" argument.

**Reply by abrokenjester** [reply]
The "node" argument is the resource (usually a blank node though it can also be an IRI) that indicates the start of the RDF Collection. See the explanation here: https://rdf4j.org/documentation/programming/model/#rdf-collections. In the diagram in that document, `_:n1` is the node that is the start of the list.

**Comment by hmottestad** [comment]
Hi Tore. 

This isn't related to your question. I just wanted to say that I'm a developer on RDF4J here in Norway and I'm always excited to hear about others working with graph and semantic technologies here in Norway. 

Any chance you could share some info about the project you're working on?

Mvh. 
Håvard Ottestad

**Reply by toretest** [reply]
At my work at statnett through my component techpros.no. Use rdf4j kotlin and graphdb to start binding different data sources not direct related.
